src/components/v5/common/ActionSidebar/hooks.tsx

```diff
@@ -453,3 +453,18 @@ export const useActionFormBaseHook: UseActionFormBaseHook = ({
     getFormOptions(undefined, form);
   });
 };
+
+export const useCloseSidebarClick = (dirtyFields: Record<string, any>) => {
```

Comment:
I think you can get `dirtyFields` from `useForm`

Reply:
I've used `useFormContext` and it's working fine